README.md

```diff
@@ -1,51 +1,74 @@
 [![Latest Release](https://img.shields.io/github/release/metabase/metabase.svg?label=latest%20release)](https://github.com/metabase/metabase/releases)
 [![GitHub license](https://img.shields.io/badge/license-AGPL-05B8CC.svg)](https://raw.githubusercontent.com/metabase/metabase/master/LICENSE.txt)
 [![Circle CI](https://circleci.com/gh/metabase/metabase.svg?style=svg&circle-token=3ccf0aa841028af027f2ac9e8df17ce603e90ef9)](https://circleci.com/gh/metabase/metabase)
 [![Leiningen Dependencies Status](https://jarkeeper.com/metabase/metabase/status.svg)](https://jarkeeper.com/metabase/metabase)
 [![NPM Dependencies Status](https://david-dm.org/metabase/metabase.svg)](https://david-dm.org/metabase/metabase)
 
-# Overview
-
-Metabase is an easy way to generate charts and dashboards, ask simple ad hoc queries without using SQL, and see detailed information about rows in your Database. You can set it up in under 5 minutes, and then give yourself and others a place to ask simple questions and understand the data your application is generating. It is not tied to any specific framework and can be used out of the box with minimal configuration.
-
-With a bit of tagging and annotation of what the tables and fields in your database mean, it can be used to provide a rich, humanized version analytics server and administration interface.
+# Features
+- 5 minute [setup](http://www.metabase.com/docs/latest/setting-up-metabase) (We're not kidding)
+- Let anyone on your team [ask questions](http://www.metabase.com/docs/latest/users-guide/03-asking-questions) without knowing SQL
+- Rich beautiful [dashboards](http://www.metabase.com/docs/latest/users-guide/05-sharing-answers) with auto refresh and fullscreen
+- SQL Mode for analysts and data pros
+- Create canonical [segments and metrics](http://www.metabase.com/docs/latest/administration-guide/05-segments-and-metrics) for your team to use
+- Send data to Slack or email on a schedule with [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses)
+- View data in Slack anytime with [Metabot](http://www.metabase.com/docs/latest/users-guide/10-metabot)
+- [Humanize data](http://www.metabase.com/docs/latest/administration-guide/03-metadata-editing) for your team by renaming, annotating and hiding fields
 
-For more information check out [www.metabase.com](http://www.metabase.com).
+For more information check out [metabase.com](http://www.metabase.com)
 
+## Supported databases
 
-# Security Disclosure
+- Postgres
+- MySQL
+- Druid
+- SQL Server
+- Redshift
+- MongoDB
+- Google BigQuery
+- SQLite
+- H2
 
-Security is very important to us. If discover any issue regarding security, please disclose the information responsibly by sending an email to security@metabase.com and not by creating a GitHub issue.
+Don't see your favorite database? File an issue to let us know.
 
-# Installation
+## Installation
 
 You can run Metabase in two primary ways, as a docker container or as a jar.
```

Comment:
Thinking it would be nice to add a link to our installation guides here and just lead into the rest of the content here as a TLDR.  Maybe something like ...

Metabase can be run just about anywhere so checkout our [Installation Guides](http://www.metabase.com/docs/latest/operations-guide/start.html#installing-and-running-metabase) for detailed instructions.  If you're impatient like we tend to be, here's the 20 second version.